.github/workflows/devcontainer.yml

```diff
@@ -90,17 +90,14 @@ jobs:
           --env LOCAL_WORKSPACE_FOLDER=$(pwd) \
           --env PYTEST_EXTRA_OPTIONS=$PYTEST_EXTRA_OPTIONS \
           --workdir /workspaces/MLOS \
-          --name mlos-devcontainer mlos-devcontainer sleep infinity
+          --name mlos-devcontainer mlos-devcontainer sleep 1800
     - name: Fixup vscode uid/gid in the running container
       timeout-minutes: 3
       run: |
         set -x
         docker exec --user root mlos-devcontainer groupmod --non-unique -g `id -g` vscode
         docker exec --user root mlos-devcontainer usermod --non-unique -u `id -u` -g `id -g` vscode
         docker exec --user root mlos-devcontainer chown -R vscode:vscode /home/vscode
-        docker exec --user root mlos-devcontainer mkdir -p /opt/conda/pkgs/cache /var/cache/pip
-        docker exec --user root mlos-devcontainer chown -R vscode /opt/conda/pkgs/cache /var/cache/pip
-        docker exec --user root mlos-devcontainer chown -R vscode /opt/conda/envs/mlos
```

Comment:
Minor time improvement - we don't need to walk that filesystem when we create the files with conda as the group owner.